rotkehlchen/globaldb/updates.py

```diff
@@ -24,6 +23,7 @@
 from rotkehlchen.utils.misc import is_production
 from rotkehlchen.utils.network import query_file
 
+from ..db.settings import CachedSettings
```

Comment:
Again here .... don't use relative imports like this, unless it's the same module.